nixos/modules/tasks/auto-upgrade.nix

```diff
@@ -223,7 +234,7 @@ in {
         ''}
 
         if [ "''${booted}" = "''${built}" ]; then
-          ${nixos-rebuild} switch ${toString cfg.flags}
+          ${nixos-rebuild} ${toString (cfg.target)} ${toString cfg.flags}
```

Comment:
The argument is already a string, so `toString` shouldn't be necessary here.

Reply:
ok.